tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/CI/MsTestV2Tests.cs

```diff
@@ -14,19 +14,37 @@
 using Datadog.Trace.TestHelpers;
 using Xunit;
 using Xunit.Abstractions;
+#pragma warning disable SA1402
 
 namespace Datadog.Trace.ClrProfiler.IntegrationTests.CI
 {
-    public class MsTestV2Tests : TestHelper
+    public class MsTestV2Tests(ITestOutputHelper output) : MsTestV2TestsBase("MSTestTests", output);
+
+    public class MsTestV2Tests2(ITestOutputHelper output) : MsTestV2TestsBase("MSTestTests2", output);
+
+    [Collection("MsTestV2Tests")]
+    public abstract class MsTestV2TestsBase : TestHelper
     {
-        private const string TestSuiteName = "Samples.MSTestTests.TestSuite";
-        private const string TestBundleName = "Samples.MSTestTests";
+        private readonly GacFixture _gacFixture;
 
-        public MsTestV2Tests(ITestOutputHelper output)
-            : base("MSTestTests", output)
+        public MsTestV2TestsBase(string sampleAppName, ITestOutputHelper output)
+            : base(sampleAppName, output)
         {
+            TestBundleName = $"Samples.{sampleAppName}";
+            TestSuiteName = "Samples.MSTestTests.TestSuite";
```

Comment:
I think this is a typo?
```suggestion
            TestSuiteName = $"Samples.{sampleAppName}.TestSuite";
```

Reply:
Nope, I did the link thing, so both have the same namespace.